api/client/beacon/health/health.go

```diff
@@ -26,7 +24,7 @@ func (n *NodeHealthTracker) HealthUpdates() <-chan bool {
 	return n.healthChan
 }
 
-func (n *NodeHealthTracker) IsHealthy() bool {
+func (n *NodeHealthTracker) IsHealthy(_ context.Context) bool {
```

Comment:
Why add this parameter when it's ignored anyway?

Reply:
It needs to follow the interface, there's actually a bug with how we use it previously the test and the implementation work differently .